README.md

```diff
@@ -337,6 +337,42 @@ end
 
 This is an incredible powerful feature and further explained under [Smell Suppresion](docs/Smell-Suppression.md).
 
+### Generating a 'todo' list
+
+Integrating tools like `Reek` into an existing larger codebase can be daunting when you have to fix
```

Comment:
Actually I replaced all occurences of `reek` and `Reek` with Reek in our README and our docs. Boyscout rule ;)